atom/browser/api/atom_api_browser_window.cc

```diff
@@ -1082,8 +1082,9 @@ void BrowserWindow::ToggleTabBar() {
   window_->ToggleTabBar();
 }
 
-void BrowserWindow::AddTabbedWindow(NativeWindow* window) {
-  window_->AddTabbedWindow(window);
+void BrowserWindow::AddTabbedWindow(NativeWindow* window,
+  mate::Arguments* args) {
+  window_->AddTabbedWindow(window, args);
```

Comment:
We shouldn't be processing/using `native-mate` outside the api layer, can you instead change the signature of `NativeWindow::AddTabbedWindow` to have a return value and throw error in `api::BrowserWindow` ? Thanks!

Reply:
on it!